automated-deployments-using-github-actions.md

```diff
@@ -0,0 +1,300 @@
+# Automating Okteto cloud workflows using Okteto Actions and GitHub Actions.
+
+## Introduction
+
+GitHub Actions provides developers with the functionality of building and automating  the execution of their software development workflows directly in a project’s repository.
+
+In this tutorial you will learn how you can automate the workflow of an existing Flask API whose continuous integration process is managed using [GitHub Actions](https://github.com/features/actions). You will learn about the Okteto actions and how you can use them to work with Okteto from a GitHub Action build.
```

Comment:
Hm, I'm not sure what you mean by Okteto actions. Do you mean to say GitHub actions created by Okteto?

Reply:
Alright.